Hi @naglis @lasley , In this PR sentry tests fails in Travis. Did you have an idea to fix this ? Thinks. 

Here is the logs : 
```
2017-12-13 11:20:45,686 6113 INFO openerp_test odoo.addons.sentry.tests.test_client: OK
2017-12-13 11:20:45,686 6113 INFO openerp_test odoo.modules.module: odoo.addons.sentry.tests.test_logutils running tests.
2017-12-13 11:20:45,687 6113 INFO openerp_test odoo.addons.sentry.tests.test_logutils: test_cookie_as_string (odoo.addons.sentry.tests.test_logutils.TestOdooCookieSanitizer)
2017-12-13 11:20:45,687 6113 ERROR openerp_test odoo.addons.sentry.tests.test_logutils: FAIL
2017-12-13 11:20:45,689 6113 INFO openerp_test odoo.addons.sentry.tests.test_logutils: test_cookie_as_string_with_partials (odoo.addons.sentry.tests.test_logutils.TestOdooCookieSanitizer)
2017-12-13 11:20:45,689 6113 INFO openerp_test odoo.addons.sentry.tests.test_logutils: test_cookie_header (odoo.addons.sentry.tests.test_logutils.TestOdooCookieSanitizer)
2017-12-13 11:20:45,689 6113 ERROR openerp_test odoo.addons.sentry.tests.test_logutils: FAIL
2017-12-13 11:20:45,691 6113 INFO openerp_test odoo.addons.sentry.tests.test_logutils: ======================================================================
2017-12-13 11:20:45,691 6113 ERROR openerp_test odoo.addons.sentry.tests.test_logutils: FAIL: test_cookie_as_string (odoo.addons.sentry.tests.test_logutils.TestOdooCookieSanitizer)
2017-12-13 11:20:45,692 6113 ERROR openerp_test odoo.addons.sentry.tests.test_logutils: Traceback (most recent call last):
2017-12-13 11:20:45,694 6113 ERROR openerp_test odoo.addons.sentry.tests.test_logutils: `   File "/home/travis/build/OCA/server-tools/sentry/tests/test_logutils.py", line 35, in test_cookie_as_string
2017-12-13 11:20:45,695 6113 ERROR openerp_test odoo.addons.sentry.tests.test_logutils: `     m=proc.MASK,
2017-12-13 11:20:45,696 6113 ERROR openerp_test odoo.addons.sentry.tests.test_logutils: ` AssertionError: 'website_lang=en_us;session_id=hello;Session_ID=hello;foo=bar' != 'website_lang=en_us;session_id=********;Session_ID=********;foo=bar'
2017-12-13 11:20:45,698 6113 INFO openerp_test odoo.addons.sentry.tests.test_logutils: ======================================================================
2017-12-13 11:20:45,698 6113 ERROR openerp_test odoo.addons.sentry.tests.test_logutils: FAIL: test_cookie_header (odoo.addons.sentry.tests.test_logutils.TestOdooCookieSanitizer)
2017-12-13 11:20:45,699 6113 ERROR openerp_test odoo.addons.sentry.tests.test_logutils: Traceback (most recent call last):
2017-12-13 11:20:45,701 6113 ERROR openerp_test odoo.addons.sentry.tests.test_logutils: `   File "/home/travis/build/OCA/server-tools/sentry/tests/test_logutils.py", line 78, in test_cookie_header
2017-12-13 11:20:45,702 6113 ERROR openerp_test odoo.addons.sentry.tests.test_logutils: `     'a_session_id_here={m}'.format(m=proc.MASK))
2017-12-13 11:20:45,703 6113 ERROR openerp_test odoo.addons.sentry.tests.test_logutils: ` AssertionError: 'foo=bar;session_id=hello;Session_ID=hello;a_session_id_here=hello' != 'foo=bar;session_id=********;Session_ID=********;a_session_id_here=********'
2017-12-13 11:20:45,705 6113 INFO openerp_test odoo.addons.sentry.tests.test_logutils: Ran 3 tests in 0.004s
2017-12-13 11:20:45,705 6113 ERROR openerp_test odoo.addons.sentry.tests.test_logutils: FAILED
2017-12-13 11:20:45,706 6113 INFO openerp_test odoo.addons.sentry.tests.test_logutils:  (failures=2)
2017-12-13 11:20:45,706 6113 ERROR openerp_test odoo.modules.module: Module sentry: 2 failures, 0 errors
```

--- a/base_exception/migrations/10.0.1.0.0/pre-migration.py
+++ b/base_exception/migrations/10.0.1.0.0/pre-migration.py
@@ -1,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# © 2017 Akretion, Mourad EL HADJ MIMOUNE
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    cr = env.cr
+    openupgrade.rename_tables(cr, [('sale_exception', 'exception_rule')])
+    openupgrade.rename_models(cr, [('sale.exception', 'exception.rule')])

